src/Microsoft.Health.Fhir.Core/Models/SortParameterStatus.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Models
+{
+    public enum SortParameterStatus
```

Comment:
I suggest putting some comments on these to explain what thy mean (especially Supported vs Enabled)

Reply:
Done